### ISSUE TITLE
Instructions for CA Certs

### ISSUE THREAD

**Issue by cdelsignoredyer** [babun/babun#829]
Every time I've installed .babun for Windows, I have had to set up CA certs manually and manually enter it into the .babunrc file. This is frustrating and I'm sure many raw users have this as a blocker. There needs to be better documentation on how to do this.   Every bit of help I find online looks like this:

https://stackoverflow.com/questions/3160909/how-do-i-deal-with-certificates-using-curl-while-trying-to-access-an-https-url

and tells you to run

`apt-cyg install ca-certificates`

Which doesn't work. apt-cyg isn't a thing in babun. I understand that some people want to start with a raw baseline of tools, but I think this is super duper confusing for the majority of people who would be using this.

Thanks for any help you can provide.

@loopy1492

**Comment by cdelsignoredyer** [comment]
'''
{ ~ }  » curl -fsSL https://get.docksal.io | DOCKER_NATIVE=1 sh
)h: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8
curl: (60) SSL certificate problem: self signed certificate in certificate chain
'''

**Comment by loopy3025** [comment]
Yeah. All the certificates and permissions you need to set perfectly to get babun working on a windows 10 machine are a rabbit hole of google searches. :(

**Comment by loopy3025** [comment]
and pact install ca-certificates only says it's already installed. You have to point it at the folder in your config.

**Comment by izaac** [comment]
Have you guys tried to run the `update.bat` in your babun installation, that fired up the cygwin installer for me and updated the ca-certificates update.
